pages/1_14/getting_started/new_project.md

```diff
@@ -0,0 +1,164 @@
+(like this right now!) or update Forge, you should really make sure this is updated to the latest mappings version.
+You can get the latest version over at the [MCP Bot exports site](http://export.mcpbot.bspk.rs/). You should always use
+the latest stable release for your Minecraft version where possible, otherwise use the latest snapshot. To update you
+simply just need to enter the version number into the value as explained in the comments above it.
+
+The rest of the `minecraft` section contains information about the run tasks gradle will generate for your IDE. You 
+generally don't need to mess with those.
+
+### Dependencies
+
+```groovy
+dependencies {
+    minecraft 'net.minecraftforge:forge:1.14.4-28.1.26'
+
+    // Other stuff
+}
+```
+
+In this section you will add any dependencies your project may have.
+At the beggining the only thing that is added here is the forge version.
+
+If you ever want to update the version of
+Forge within your mod, simply update this value.
+
+You may put jars on which you depend on in ./libs or you may define them like so:
+```groovy
+compile "some.group:artifact:version:classifier"
+compile "some.group:artifact:version"
+```
+
+The 'provided' configuration is for optional dependencies that exist at compile-time but might not at runtime.
+```groovy
+provided 'com.mod-buildcraft:buildcraft:6.0.8:dev'
+```
+
+For more info:
+http://www.gradle.org/docs/current/userguide/artifact_dependencies_tutorial.html
+
+http://www.gradle.org/docs/current/userguide/dependency_management.html
+
+### Jar manifest
+This last section of the build.gradle allows you to add
+values to your output jar's manifest. The default
+`build.gradle` adds some values about the author and mod here.
+
+```groovy
+jar {
+    manifest {
+        attributes([
+            "Specification-Title": project.name, // This may be different by default but you can change it to project.name
+            "Specification-Vendor": "author",
+            "Specification-Version": "1", // We are version 1 of ourselves
+            "Implementation-Title": project.name,
+            "Implementation-Version": "${version}",
+            "Implementation-Vendor" :"author",
+            "Implementation-Timestamp": new Date().format("yyyy-MM-dd'T'HH:mm:ssZ")
+        ])
+    }
+}
+```
+
+As you can see the date, name, and version have been taken care of.
+All you have to do is enter your name in the places of `author`
+
+### Using the gradle.properties
+
+To make life easier, you don't have to keep searching through the build.gradle for a few values to change any time you
+want to update a version. You can make use of your `gradle.properties` file to collect variables in. You can check out
+an example of that in use here in Sparks Hammers:
+* [build.gradle](https://github.com/thebrightspark/SparksHammers/blob/1.12/build.gradle)
+* [gradle.properties](https://github.com/thebrightspark/SparksHammers/blob/1.12/gradle.properties)
```

Comment:
These examples are 1.12, even Gradle works the same some people might get confused.